.gitignore

```diff
@@ -32,7 +32,6 @@ hs_err_pid*
 target
 
 ### Eclipse template
-.metadata
```

Comment:
Why did you delete this? Did you pull master to your branch before PR?

Reply:
it's duplicated!
Have a look at line 46:

https://github.com/dcaviedesm/jazz-porque-si/blob/f5aad54c49d5eec1e165d0da88fc34caca1037d5/.gitignore#L46